src/main/java/uk/gov/hmcts/sptribs/document/DocumentConstants.java

```diff
@@ -10,6 +10,8 @@ public final class DocumentConstants {
     public static final String SPTRIBS_DRAFT_APPLICATION_DOCUMENT_NAME = "draft-sptribs-application-";
 
     public static final String DOCUMENT_ID = "documentId";
+    public static final String DOCUMENT_NOT_SELECTED_MESSAGE = "A document was not selected for upload. "
+        + "Please select a document to upload or press the 'Remove' button";
```

Comment:
Has the error message text been confirmed by Linda and/or Jon?

Reply:
Yes by Linda